dist.ini

```diff
@@ -23,6 +23,9 @@ bugtracker.web  = https://github.com/pjf/ipc-system-simple/issues
 [AutoPrereqs]
 skip = BSD::Resource
 
+[Prereqs]
+Win32::ShellQuote = 0
```

Comment:
Not sure how one would go about requiring Win32::ShellQuote only if the OS is Windows.

Reply:
Oh, cool, I wasn't aware of this module. Done in a1fff1b.